save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/vuln/CreateVulnerabilityView.kt

```diff
@@ -301,17 +301,17 @@ val createVulnerabilityView: VFC = VFC {
                                             required = true
                                             defaultValue = 0
                                             onChange = { event ->
-                                                val progress = event.target.value.toIntOrNull()
-                                                if (progress !in (1..100)) {
-                                                    setConflictErrorMessage("Progress must be an integer number from 1 to 100")
+                                                val progress = event.target.value.toFloatOrNull()
+                                                if (progress!! !in (0f..10f)) {
```

Comment:
not sure if Float Range will work, hope you have tested it

Reply:
Yes. Tested it